expression/chunk_executor.go

```diff
@@ -368,3 +368,42 @@ func VectorizedFilter(ctx sessionctx.Context, filters []Expression, iterator *ch
 	}
 	return selected, nil
 }
+
+// VectorizedFilterConsiderNull applies a list of filters to a Chunk and
+// returns two bool slices, `selected` indicates whether a row passed the
+// filters, `isNull` indicates whether the result of the filter is null.
+// Filters is executed vectorized.
+func VectorizedFilterConsiderNull(ctx sessionctx.Context, filters []Expression, iterator *chunk.Iterator4Chunk, selected []bool, isNil []bool) ([]bool, []bool, error) {
```

Comment:
1. s/isNil/isNull/
2. Can we refactor `VectorizedFilter()` to call `VectorizedFilterConsiderNull()`?